hyundai_palisade_2023.dbc

```diff
@@ -0,0 +1,1088 @@
+VERSION ""
+
+
+NS_ :
+ NS_DESC_
+ CM_
+ BA_DEF_
+ BA_
+ VAL_
+ CAT_DEF_
+ CAT_
+ FILTER
+ BA_DEF_DEF_
+ EV_DATA_
+ ENVVAR_DATA_
+ SGTYPE_
+ SGTYPE_VAL_
+ BA_DEF_SGTYPE_
+ BA_SGTYPE_
+ SIG_TYPE_REF_
+ VAL_TABLE_
+ SIG_GROUP_
+ SIG_VALTYPE_
+ SIGTYPE_VALTYPE_
+ BO_TX_BU_
+ BA_DEF_REL_
+ BA_REL_
+ BA_DEF_DEF_REL_
+ BU_SG_REL_
+ BU_EV_REL_
+ BU_BO_REL_
+ SG_MUL_VAL_
+
+BS_:
+
+BU_: IAP ODS _4WD BCM HUD DATC MDPS AAF_Tester AEMC SMK _4WD EPB CUBIS MTS TMU EVP CGW TPMS LPI DI_BOX SPAS EMS LCA TCU IBOX FATC AFLS FPCM SCC AHLS AVM ABS SNV OPI PGS SAS AAF Dummy LDWS_LKAS LVR ESC PSB CLU ECS ACU REA
+
+BO_ 67 DATC13: 8 XXX
+ SG_ CF_Datc_AcDisp : 22|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_AqsDisp : 20|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_AutoDefogBlink : 16|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_AutoDisp : 12|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_ChgReqDisp : 8|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_ClmScanDisp : 18|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_DualDisp : 30|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_FrDefLed : 14|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_FrontBlwDisp : 60|4@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_IntakeDisp : 10|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_IonClean : 6|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_ModDisp : 2|4@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_OpSts : 25|3@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_PSModDisp : 56|4@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_PwrInf : 32|4@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearAutoDisp : 40|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearBlwDisp : 52|4@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearChgReqDisp : 46|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearClimateScnDisp : 44|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearManual : 38|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearModDisp : 48|4@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_RearOffDisp : 42|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Datc_TempDispUnit : 0|2@1+ (1,0) [0|0] "" XXX
+ SG_ CF_Mtc_MaxAcDisp : 28|2@1+ (1,0) [0|0] "" XXX
+
+BO_ 80 LKAS: 16 XXX
+ SG_ CHECKSUM : 0|16@1+ (1,0) [0|0] "" XXX
+ SG_ COUNTER : 16|8@1+ (1,0) [0|0] "" XXX
+ SG_ FCA_SYSWARN : 40|1@0+ (1,0) [0|0] "" XXX
+ SG_ LFA_BUTTON : 56|1@1+ (1,0) [0|0] "" XXX
+ SG_ LKA_ASSIST : 62|1@1+ (1,0) [0|0] "" XXX
+ SG_ LKA_ICON : 38|2@1+ (1,0) [0|0] "" XXX
+ SG_ LKA_MODE : 24|3@1+ (1,0) [0|0] "" XXX
+ SG_ LKA_WARNING : 32|1@1+ (1,0) [0|0] "" XXX
+ SG_ NEW_SIGNAL_1 : 27|2@1+ (1,0) [0|0] "" XXX
+ SG_ NEW_SIGNAL_2 : 70|2@0+ (1,0) [0|0] "" XXX
+ SG_ NEW_SIGNAL_3 : 111|8@0+ (1,0) [0|0] "" XXX
+ SG_ SET_ME_1 : 80|1@0+ (1,0) [0|0] "" XXX
+ SG_ STEER_MODE : 65|3@1+ (1,0) [0|0] "" XXX
+ SG_ STEER_REQ : 52|1@1+ (1,0) [0|0] "" XXX
+ SG_ TORQUE_REQUEST : 41|11@1+ (1,-1024) [0|0] "" XXX
```

Comment:
This looks identical to LKAS in hyundai_canfd, let's make a base CAN FD DBC and put this in there.